src/main/conversions.c

```diff
@@ -1032,6 +1046,10 @@ as_status pyobject_to_record(AerospikeClient *self, as_error *err,
                 char *str = PyBytes_AsString(value);
                 as_bytes_set(bytes, 0, (const uint8_t *)str, str_len);
 
+                if (strstr(value->ob_type->tp_name, "HyperLogLog")) {
+                    bytes->type = AS_BYTES_HLL;
+                }
```

Comment:
As unlikely as it is, this might catch a user's object with `HyperLogLog` in the name if it inherits from bytes. I think using the full name would be better.